### PR TITLE
fix RPC call application.exit.later unsuccessful: connection not open

### LINKED ISSUE #139: RPC call application.exit.later unsuccessful: connection not open
Hey guys, I've encountered a problem. When we call the destroy method of SmartConnect, the cleanUp function within it initiates a session.close operation first. This results in an error when the destroy method of WebsocketConnection is called immediately afterwards, as model.session.call('application.exit.later', [timeout]) is called on a session that's already been closed.
Besides, could you help explain the function of 'application.exit.later' to me?

```javascript
{code: -32099, message: 'RPC call application.exit.later unsuccessful: connection not open'}
```

### PR DESCRIPTION
**Pull request by jinjilynn**
fix #139

### CODE REVIEW

**Comment by jourdain** [comment]
Thx it looks great.

File: js/src/SmartConnect/index.js [hunk 1]
```diff
@@ -134,8 +134,9 @@ function smartConnect(publicAPI, model) {
     return session;
   };
 
-  function cleanUp() {
+  function cleanUp(timeout) {
     if (session) {
+      timeout > 0 && session.call('application.exit.later', [timeout]);
```
Comment (jourdain) [inline js/src/SmartConnect/index.js:139]: Can you rewrite that as a if?

```js
if (timeout > 0) {
  session.call('application.exit.later', [timeout])
}
```

Comment (jourdain) [reply js/src/SmartConnect/index.js:139]: Also can you rework your commit (maybe via force push) to be 

`fix(SmartConnect): Allow delete to call exit later`

Comment (jinjilynn) [reply js/src/SmartConnect/index.js]: ok, done

**Pull request merged**